docs/azure/sdk/snippets/dependency-injection/WebApplicationBuilder/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿#region snippet_WebApplicationBuilder
+using Azure.Identity;
+using Microsoft.Extensions.Azure;
+
+WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddAzureClients(clientBuilder =>
+{
+    clientBuilder.AddSecretClient(new Uri("<key_vault_url>"));
+    clientBuilder.AddBlobServiceClient(new Uri("<storage_url>"));
+    clientBuilder.UseCredential(new DefaultAzureCredential());
+});
+
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+WebApplication app = builder.Build();
+#endregion snippet_WebApplicationBuilder
+
+if (app.Environment.IsDevelopment())
+{
+    app.UseSwagger();
```

Comment:
Why is this specific to dev environments? Why are we using these APIs in general?

Reply:
This code was included in the VS project template I used. No other reason. Only the code within the region is embedded in the accompanying doc.

Reply:
In that case, could we remove lines 14 and 15?

Reply:
Done